### ISSUE TITLE
Support Rails 5

### ISSUE THREAD

**Issue by ronen** [SchemaPlus/schema_plus#229]
Creating the issue...

...if anybody out there in github-land is interested in working on Rails 5 support for SchemaPlus, please let me know. I myself am a bit swamped these days so would welcome a collaborator stepping in. 
(I haven't even looked into whether this would be hard or trivial.)

Thanks!

[This is a duplicate of SchemaPlus/schema_plus_core#9 which is where presumably most of the work would be]

**Comment by wangaguo** [comment]
I need schema_plus for rails 5 please.

**Comment by emptyflask** [comment]
Now that SchemaPlus/schema_plus_core#9 is merged, is the rest of this basically just updating dependencies, or is there a lot more work to be done in the other schema_plus gems?

**Comment by boazy** [comment]
Unfortunately, every gem I've went through so far had to be updated individually, but that doesn't mean all of them will.

**Comment by boazy** [comment]
## Status update ##

The following gems now pass all tests with AR5:

* schema_plus_core
* schema_plus_indexes
* schema_plus_foreign_keys
* schema_plus_columns
* schema_associations

**Comment by jimcavoli** [comment]
I've opened up a PR with the changes for schema_plus_enums as well at https://github.com/SchemaPlus/schema_plus_enums/pull/6 - it has one outstanding question over how the versioning should continue before merging.

**Comment by edwardmp** [comment]
@myabc
Can you share your progress on schema_plus_pg_indexes? I kind of need it it work with AR 5 so I might take a stab at it

**Comment by ACPK** [comment]
@edwardmp - Any update now that schema_plus_pg_indexes supports Rails5?

**Comment by edwardmp** [comment]
@ACPK Ah great someone else did that. It seems that it's just schema_plus_views needs to be updated for AR 5.1 before the schema_plus gem itself can be updated.

**Comment by ronen** [comment]
Yes, the `schema_plus` gem itself can only be updated once all the individual gems have been updated.  

But as per the README, "For the most part we recommend skipping [`schema_plus`] and directly including the specific feature gems you're interested in using."   

So if the specific gems you need are already at 5.1 you can just require them directly and not bother waiting for `schema_plus`.  

(...and if any gems you need are not yet ready for 5.1, any help to get them there would be greatly appreciated!)
